src/components/fieldText/fieldText.module.scss

```
@@ -193,6 +193,11 @@
   left: 0;
   pointer-events: none;
   white-space: nowrap;
+
+  .asterisk {
+    @include field-asterisk();
```

Stylelint (17.14.1)

[error] Unexpected parentheses in argumentless mixin "field-asterisk" call (scss/at-mixin-argumentless-call-parentheses)

(scss/at-mixin-argumentless-call-parentheses)